### PR TITLE
Fix Sequelize migration issue

### PR DESCRIPTION
**Pull request by kennyshittu**
Sequelize migration scripts are not being processed on start up. This PR fixes that.

### CODE REVIEW

File: server/migrations/20180727122128-add-refresh-token-column-to-user-model.js [hunk 1]
```diff
@@ -3,15 +3,15 @@
 module.exports = {
   up: (queryInterface, Sequelize) => {
     queryInterface.addColumn(
-      'User',
+      'Users',
```
Comment (mariechatfield) [inline server/migrations/20180727122128-add-refresh-token-column-to-user-model.js:6]: Why is this changed? It looks like a migration from 2018, so I would expect it would have been run once then.

Comment (kennyshittu) [reply server/migrations/20180727122128-add-refresh-token-column-to-user-model.js:6]: @mariechatfield The old migration files were only relevant as I developed locally, at the time of deployment to staging and prod the models were up to date and we didn't have a  need to run it.

Comment (mariechatfield) [reply server/migrations/20180727122128-add-refresh-token-column-to-user-model.js:6]: That makes sense that staging and prod don't need to run any migrations, but I'm still confused about why we need to change the name of a column that was added in 2018. If the original column used the wrong name, I would expect we'd need a separate migration that removed `User` and added `Users`

File: server/configs/config.json [hunk 2]
```diff
@@ -6,13 +6,10 @@
     "dialect": "postgres",
     "options": {
       "host": "127.0.0.1",
-      "dialect": "postgres",
-      "operatorsAliases": {
-        "$ne": "Op.ne"
-      }
```
Comment (katz12) [inline server/configs/config.json:12]: Why does this need to be removed?

File: server/migrations/20180727122128-add-refresh-token-column-to-user-model.js (hunk 1 above)
Comment (katz12) [reply server/migrations/20180727122128-add-refresh-token-column-to-user-model.js:6]: I am also confused about why this change is relevant. 

Is it that this migration has never been run in staging/prod and it was incorrect?

File: package.json [hunk 3]
```diff
@@ -34,9 +34,10 @@
   "scripts": {
     "precommit": "./pre-commit.sh",
     "start": "node --trace-warnings ./server/bin/www",
-    "dev": "yarn run heroku-prebuild && yarn run start",
+    "dev": "yarn run heroku-prebuild && yarn run migration && yarn run start",
```
Comment (katz12) [inline package.json:37]: Does this only result in migration being run locally? Or do we use `dev` for the staging/prod deploys?

**Review by katz12 — COMMENTED** [review]
I know very little about sequelize, so I left comments on the parts that seemed foreign to me. Otherwise these changes make sense.

File: server/configs/config.json (hunk 2 above)
Comment (kennyshittu) [reply server/configs/config.json:12]: String based operators are now deprecated in sequelize and we don't actually use it.

File: package.json (hunk 3 above)
Comment (kennyshittu) [reply package.json:37]: Good point, Heroku uses `start`

Comment (kennyshittu) [reply package.json]: Updated

File: server/migrations/20180727122128-add-refresh-token-column-to-user-model.js (hunk 1 above)
Comment (kennyshittu) [reply server/migrations/20180727122128-add-refresh-token-column-to-user-model.js:6]: @katz12 @mariechatfield Ah, we never ran the migration scripts that's why the typo went unnoticed. I added them as I built locally but we never ran them.

**Review by katz12 — APPROVED** [review]
